package.json

```diff
@@ -37,33 +42,35 @@
     ]
   },
   "devDependencies": {
-    "babel-cli": "6.x",
-    "babel-core": "6.x",
-    "babel-preset-shopify": "15.0.1",
-    "eslint": "^3.19.0",
-    "eslint-index": "^1.3.0",
+    "babel-cli": "^6.24.1",
+    "babel-core": "^6.25.0",
+    "babel-preset-shopify": "^16.1.0",
+    "eslint": "^4.3.0",
+    "eslint-index": "^1.4.0",
     "eslint-plugin-shopify": "file:.",
-    "isparta": "4.x",
-    "istanbul": "0.4.4",
-    "mocha": "3.x"
+    "isparta": "^4.0.0",
+    "istanbul": "^0.4.5",
+    "mocha": "^3.4.2",
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1"
   },
   "peerDependencies": {
-    "eslint": "<4 >=3.17"
+    "eslint": "<5 >=4.3.0"
   },
   "dependencies": {
     "babel-eslint": "^7.2.3",
-    "eslint-plugin-ava": "^4.2.0",
-    "eslint-plugin-babel": "^4.1.1",
-    "eslint-plugin-chai-expect": "1.1.x",
+    "eslint-plugin-ava": "^4.2.1",
+    "eslint-plugin-babel": "^4.1.2",
+    "eslint-plugin-chai-expect": "^1.1.1",
     "eslint-plugin-flowtype": "^2.32.1",
-    "eslint-plugin-import": "2.2.x",
+    "eslint-plugin-import": "^2.2.0",
```

Comment:
Not a blocker for this PR, but `shopify/shopify` is going to fail `yarn check` until a new `eslint-plugin-import` release ([it will loosen up the expected dependencies](https://github.com/benmosher/eslint-plugin-import/commit/72d13858e044569d6f194922e507578410601f28)).

Seeing:
```
error "\u001b[2meslint-plugin-shopify#\u001b[22meslint-plugin-import#eslint@2.x - 3.x" doesn't satisfy found match of "eslint@4.3.0"
```